src/Germany/Mews.Fiscalizations.Germany/FiskalyClient.cs

```diff
@@ -32,16 +31,22 @@ public Task<ResponseResult<Model.Client>> GetClientAsync(AccessToken token, Guid
 
         public async Task<ResponseResult<Model.Client>> CreateClientAsync(AccessToken token, Guid tssId)
         {
-            var tss = await GetTssAsync(token, tssId);
-            var tssCertificate = tss.SuccessResult.Certificate;
-            var certificate = new X509Certificate2(Encoding.UTF8.GetBytes(tssCertificate));
-            var serialNumber = certificate.GetCertHashString();
-
-            var request = RequestCreator.CreateClient(serialNumber);
+            var id = Guid.NewGuid();
```

Comment:
So looks like this should be an optional parameter?

Reply:
![image](https://user-images.githubusercontent.com/51375082/134347943-690c98f7-fa15-46c7-876b-4a5723b7bbcc.png)

Reply:
Not really sure what you're trying to say? 
I'm saying that the code that wants to create a client should be capable of defining the ID of the client. Instead of you generating a random one.

Reply:
I was trying to answer your question but seems like i misunderstood it.

Yes, this can be optional (anyway we return the ID in the response).

Reply:
it just looks weird that u create something and u have to provide the ID for it 😄 
and since we return the ID in the response, then there is no benefit from adding the ID in the param as nullable.

Reply:
But we want it for the TSS ID (as per the other comment) so why not allow it for the client too.